src/index.js

```diff
@@ -1,6 +1,10 @@
 import './style.css';
-import { submit, name, score } from './modules/const.js';
-import { addScore, renderScore, showList } from './modules/renderScore.js';
+import {
+  submit, name, score, refresh,
+} from './modules/const.js';
+import {
+  addScore, refreshScore, renderScore, showList,
+} from './modules/renderScore.js';
 
 window.addEventListener('load', () => {
   showList();
```

Comment:
- On page load, your app doesn't doesn't populate the UI with the expected data. A likely cause for this is that your `showList` function doesn't fetch data from the API before rendering the scores. I advise that you fix this 🙏 🙏 

Your project:
<img width="953" alt="2ff3f" src="https://user-images.githubusercontent.com/73607512/208951508-4d56670c-be98-4c35-a7da-f74b558b4719.PNG">

Reply:
@mckent05 I think its supposed to be like that as stated in the curriculum that only when the refresh button is clicked then it loads from the api

Reply:
@mckent05 I would appreciate a reply as i think the changes is not supposed to be so.

Reply:
- The refresh button should update the UI after a new score is submitted, on page load/reload already existing scores in the API should be populated on the scoreboard. Not to worry, You can ask for a [second opinion](https://airtable.com/shrQAqnBwek5a0O0s) review if you still do not agree with mine, a senior CR would approve it or otherwise 😄 😄